lms/djangoapps/bookmarks/tests/test_api.py

```diff
@@ -178,3 +211,5 @@ def test_delete_bookmark_raises_error(self):
         """
         with self.assertRaises(ObjectDoesNotExist):
             api.delete_bookmark(user=self.other_user, usage_key=self.vertical.location)
+
+        self.assert_no_events_were_emitted()
```

Comment:
Is this actually going to pass?  It seems that the event is currently emitted before the delete is attempted, so we would get an event whether it succeeded or failed.